fxevent/event.go

```diff
@@ -98,20 +88,15 @@ type StopSignal struct{ Signal os.Signal }
 // StopError is emitted whenever we fail to stop cleanly.
 type StopError struct{ Err error }
 
-// Rollback is emitted whenever a service fails to start.
-type Rollback struct{ StartErr error }
+// RollingBack is emitted whenever a service fails to start.
+type RollingBack struct{ StartErr error }
 
 // RollbackError is emitted whenever we fail to rollback cleanly after
 // a start error.
 type RollbackError struct{ Err error }
 
-// Running is emitted whenever an application is started successfully.
-type Running struct{}
-
-// CustomLoggerError is emitted whenever a custom logger fails to construct.
-type CustomLoggerError struct{ Err error }
-
-// CustomLogger is emitted whenever a custom logger is set.
+// CustomLogger is emitted whenever a custom logger is set or produces an error.
```

Comment:
Can we change this to a verb form as well? Maybe `CustomLoggerSet`?

Reply:
I think it's an antipattern to add `Set` or `Get` in function names so I don't like that in structs either. In any case, I think this is redundant anyway since this is a generic event.

Reply:
A small nit on the documentation wording: as someone that doesn't have a lot of context, it's not clear that the the "produced error" mentioned here is related to setting the custom logger. maybe something like: 
```
// CustomLogger is emitted whenever a custom logger is set or an error is produced in the process.
```

Reply:
I suppose we can stick with CustomLogger for now, and change it up in the future PR where we're adding/changing events.